dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -458,6 +453,11 @@ void SlashCommandHandler::HandleChatCommand(const std::u16string& command, Entit
 
 		GameMessages::SendToggleGMInvis(entity->GetObjectID(), false, UNASSIGNED_SYSTEM_ADDRESS); // need to retoggle because it gets reenabled on creation of new character
 	}
+	
+	if ((chatCommand == "playanimation" || chatCommand == "playanim") && args.size() == 1 && entity->GetGMLevel() >= GAME_MASTER_LEVEL_DEVELOPER) {
+		std::u16string anim = GeneralUtils::ASCIIToUTF16(args[0], args[0].size());
```

Comment:
Ugh, another case where we shouldn't use this function on user input.

Reply:
> Another case where we shouldn't use this function on user input.

Yeah... there's a ton of these.